### PR TITLE
The parallel process with & symbol not supported in windows. use npm-…

### LINKED ISSUE #1083: External contributor instructions don't work for web on Windows
Hi, I've followed the instructions in your README [here](https://github.com/Expensify/Expensify.cash#running-the-apps-via-production-api-proxy-contributors-) and have set everything up fresh. When I run `npm run proxy` I see this in CLI:
```
> expensify.cash@1.0.1-292 proxy ...\Expensify.cash
> node proxy.js & webpack-dev-server --open --config config/webpack/webpack.dev.js --env.proxy=true

Proxy server listening at http://localhost:9000
```

Navigating to `http://localhost:9000` redirects me to expensify.com. I've followed all the steps (though if I'm supposed to set up ngrok, the linked stackoverflow is private)

Please let me know how to set up the project as an external contributor. Thank you :)

(PS: on Windows 10, Firefox and Edge Chromium)

### PR DESCRIPTION
**Pull request by wildan-m**
The parallel process with & symbol not supported in windows. use npm-run-all instead.

Resolve `Cannot assign to read only property 'exports' of object '#<Object>'` error in windows 10.

### Details
We sometimes use & to run multiple command in parallel, but cmd.exe (npm run-script uses it by default) does not support the &. Half of Node.js users are using it on Windows, so the use of & might block contributions. npm-run-all --parallel works well on Windows as well.

### Fixed Issues
Fixes #1083

### Tests
1. clone https://github.com/Expensify/Expensify.cash/ in windows 10
2. execute npm run install
3. copy .env.production to .env
4. change USE_WEB_PROXY=false to USE_WEB_PROXY=true
5. Execute `npm run web`

### Tested On

- [x] Web
- [x] Mobile Web

#### Web
image

#### Mobile Web
no changes

#### Desktop
no changes

#### iOS
no changes

#### Android
no changes

### CODE REVIEW

**Comment by dylmye** [comment]
I can confirm this branch works for me! 🎉

File: babel.config.js [hunk 1]
```diff
@@ -16,10 +16,12 @@ const webpack = {
         production: {
             presets: defaultPresets,
             plugins: [...defaultPlugins, 'transform-remove-console'],
+            sourceType: 'unambiguous',
         },
         development: {
             presets: defaultPresets,
             plugins: defaultPlugins,
+            sourceType: 'unambiguous',
```
Comment (marcaaron) [inline babel.config.js:24]: Why this change? Shouldn't this be `module` which is the default?

https://babeljs.io/docs/en/options#sourcetype

**Review by marcaaron — COMMENTED** [review]
This proposal seems reasonable to me. Feel free to merge if it tests well for you @timszot

File: babel.config.js (hunk 1 above)
Comment (wildanmuhlis2) [reply babel.config.js:24]: Hi @marcaaron ,

I've got the solution from this link:

https://stackoverflow.com/a/56283408/1174657

In Win 10, this is what happened without unambiguous sourceType:

```
global.js:20 Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
    at Module.<anonymous> (global.js:20)
    at Module../node_modules/webpack/buildin/global.js (app-3177bbb0b1e5efe731e1.bundle.js:159668)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Module../node_modules/sockjs-client/dist/sockjs.js (app-3177bbb0b1e5efe731e1.bundle.js:152551)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Object../node_modules/webpack-dev-server/client/clients/SockJSClient.js (SockJSClient.js:28)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
```

This link state that this error occurred when we are trying to use CommonJS while we need to use ES modules.

The alternative solution is by replacing 

```
module.exports = myfunction
```

with

```
export default myfunction
```

But I'm not sure which code to replace.

https://flaviocopes.com/cannot-assign-readonly-property-export/

Comment (marcaaron) [reply babel.config.js:24]: My guess is that it's the export at the bottom of this file and we can maybe change this to

```js
export default ({caller}) => {
    // For `react-native` (iOS/Android) caller will be "metro"
    // For `webpack` (Web) caller will be "@babel-loader"
    const runningIn = caller(({name}) => name);
    return ['metro', 'babel-jest'].includes(runningIn) ? metro : webpack;
}
```

**Comment by timszot** [comment]
@wildanmuhlis2 Please resolve conflicts that have arisen in this PR.

**Comment by wildanmuhlis2** [comment]
@timszot Resolved

**Review by timszot — APPROVED** [review]
LGTM, leaving merge for you @marcaaron since you have an unresolved comment.

File: babel.config.js (hunk 1 above)
Comment (marcaaron) [reply babel.config.js:24]: Ok I think in this case the best thing to do would be to leave a comment about why we need `sourceType: unambiguous` so that it is not altered in the future.

Many of our engineers and contributors use macOS so it is very likely that this will be accidentally removed without documentation. What do you say? Thanks.

**Comment by marcaaron** [comment]
This has been open for a while without an update so I'm going to close it. Please feel free to open a new PR or reopen if we disagree with this outcome.
